homework/unique_ptr/Tests/tests.cpp

```diff
@@ -107,3 +120,10 @@ TEST(unique_ptr, move_ctor_with_ptr) {
     EXPECT_EQ(testedSmartPtr.get(), ptr);
     EXPECT_EQ(otherSmartPtr.get(), nullptr);
 }
+
+TEST(unique_ptr, star_operator) {
+    auto ptr = new int{5};
+    auto smartPtr = UniquePointer<int>(ptr);
+
+    EXPECT_EQ(*ptr, *smartPtr.get());
```

Comment:
It's not testing 'UniquePointer :operator*()' because this should work without 'get'.

Reply:
Why should it work without `get()`? I don't know how to do it in a different way.

Reply:
Because operator* should return T& type not UniquePointer&.